index.d.ts

```diff
@@ -26,6 +26,7 @@ export const BAD_AUTHENTICATION: string;
 export const BAD_JSON: string;
 export const BAD_MSG: string;
 export const BAD_REPLY: string;
+export const BAD_CALLBACK: string;
```

Comment:
Not sure if you are okay with introducing a new error code, but since I added a check for the type of the fourth argument, I tried to be consistent with the other validation checks.